Bottom: 6b7978c581f688e54a08929b71ec9a48e91f01ba
Top:    4c10a1cc8b0bc70dafd75bacd16166aa4d8a0e5a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 00:36:27 +0100

Comment out check for GUI HID when considering hidenames

Yes - I want to hide them for a print!


---

diff --git a/src/draw.c b/src/draw.c
index 792183e..5e97177 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -184,7 +184,7 @@ DrawStrippedText (ElementTypePtr Element, int min_width)
 static void
 draw_element_name (ElementType *element)
 {
-  if ((TEST_FLAG (HIDENAMESFLAG, PCB) && gui->gui) ||
+  if ((TEST_FLAG (HIDENAMESFLAG, PCB) /*&& gui->gui*/) ||
       TEST_FLAG (HIDENAMEFLAG, element))
     return;
   if (doing_pinout || doing_assy)
